package.json

```diff
@@ -69,5 +73,17 @@
     "qrcode": "1.4.1",
     "sjcl": "1.0.8",
     "typescript": "2.9.2"
+  },
+  "lint-staged": {
+    "src/**/*.{ts,tsx}": [
+      "tslint --fix",
+      "prettier --write",
+      "git add"
+    ]
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
```

Comment:
Jeżeli będą błędy niemożliwe do autofixu, to nie dojdzie do commita?

Reply:
Nie, wtedy nie dojdzie. Trzeba wtedy poprawić ręcznie, dostajemy loga z wszystkimi błędami. Jeśli się nie da, to można dac flagę `--no-verify` i olać błąd albo można dać  komentarz`tslint-ignore`.